site/pages/platforms/next.mdx

```diff
@@ -219,4 +219,27 @@ export const GET = handle(app)
 export const POST = handle(app) 
 ```
 
+### Bonus: Page co-location
+
+If a user shares a link to your website, we may want to render the frame in the page route
+itself, instead of forcing him to share a link to the frog api route.
+
+In the example below, when a user shares the `/` (= `/page.tsx` in your Next.js App)
+path of the website on Warpcast, they will be able to see the frame.
```

Comment:
```suggestion
### Bonus: Page co-location

If a user shares a link to your website, we may want to render the frame in the page route
itself, instead of forcing them to share a link to the Frog API route.

In the example below, when a user shares the `/` (= `/page.tsx` in your Next.js App)
path of the website on Warpcast, they will be able to see the frame. This leverages the [`generateMetadata`](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#generatemetadata-function) function built-in to Next.js
```